src/models/Event.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+
+const eventSchema = new mongoose.Schema(
```

Comment:
Is this model representing what Victoria's mockup calls an "Activity"?

Reply:
I am not sure about the 'Activity' idea. If each of the events includes separate activities, I'll implemented a separate model for that.